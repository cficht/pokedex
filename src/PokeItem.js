import React, { Component } from 'react';
import './App.css';
// import data from './data.js';

export default class PokeItem extends Component {
    render() {
        return(
            <li className="poke-item">
                <h2><span className="name-case">{this.props.pokemon.pokemon}</span></h2>
                <img src={this.props.pokemon.url_image} alt=""></img>
                <h5>Type: <span className="name-case">{this.props.pokemon.type_1}</span></h5>
                <h5>Attack: {this.props.pokemon.attack}</h5>
                <h5>Defense: {this.props.pokemon.defense}</h5>
            </li>
        );
    }
}