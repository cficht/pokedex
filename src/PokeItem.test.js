import React from 'react';
import PokeItem from './PokeItem';
import renderer from "react-test-renderer";
import data from './data.js';

test('renders PokeItem.js correctly', () => {
  const tree = renderer
  .create(<PokeItem pokemon={data} />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});