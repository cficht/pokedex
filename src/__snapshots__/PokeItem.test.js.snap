// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PokeItem.js correctly 1`] = `
<li
  className="poke-item"
>
  <h2>
    <span
      className="name-case"
    />
  </h2>
  <img
    alt=""
  />
  <h5>
    Type: 
    <span
      className="name-case"
    />
  </h5>
  <h5>
    Attack: 
  </h5>
  <h5>
    Defense: 
  </h5>
</li>
`;
